test/unit/SourceLocator/SourceStubber/ReflectionSourceStubberTest.php

```diff
@@ -112,6 +115,34 @@ public function testClassStub(): void
         self::assertNull($stubData->getExtensionName());
     }
 
+    /**
+     * @requires PHP >= 8.1
+     */
+    public function testPureEnumStub(): void
+    {
+        require __DIR__ . '/../../Fixture/EnumPureForSourceStubber.php';
+
+        $stubData = $this->stubber->generateClassStub(EnumPureForSourceStubber::class);
+
+        self::assertNotNull($stubData);
+        self::assertStringEqualsFile(__DIR__ . '/../../Fixture/EnumPureForSourceStubberExpected.php', $stubData->getStub());
+        self::assertNull($stubData->getExtensionName());
+    }
+
+    /**
+     * @requires PHP >= 8.1
+     */
+    public function testBackedEnumStub(): void
+    {
+        require __DIR__ . '/../../Fixture/EnumBackedForSourceStubber.php';
```

Comment:
Note: these should be `require_once`, since phpunit tests can be repeated multiple times by tooling

Reply:
https://github.com/Roave/BetterReflection/pull/856/commits/02feabebcd8a74f76f1d8762b5ffcf253bb4a5b5